addons/mail/static/src/core/common/message_in_reply.js

```diff
@@ -22,6 +22,11 @@ export class MessageInReply extends Component {
         ) {
             return url("/mail/static/src/img/email_icon.png");
         }
-        return this.props.message.parentMessage.author.avatarUrl;
+
+        if (this.props.message.parentMessage.author) {
+            return this.props.message.parentMessage.author.avatarUrl;
+        }
+
+        return '/mail/static/src/img/smiley/avatar.jpg'
```

Comment:
Better use `DEFAULT_AVATAR` (https://github.com/odoo/odoo/blob/saas-17.1/addons/mail/static/src/core/common/persona_service.js#L9) instead of a hard-coded string.